### PR TITLE
Fix/deep link user info

### PR DESCRIPTION
**Pull request by johnxnguyen**
### Issues

- Left footer action was displaying _Create group_ instead of _Open conversation_.
- Right footer action menu visible/showing incorrect menu items

### Causes

The `SingleParticipantFragment` is not aware of whether it was opened by a deep link.

### Solutions

Check if coming from a deep link and update the footer according to the spec.

### Known Bug

When opening user profile for a connected guest user, open the menu and block, then unblock the user. The menu icon disappears, but reappears after a few seconds.

#### APK
[Download build #12503](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12503/artifact/build/artifact/wire-dev-PR2066-12503.apk)
[Download build #12504](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12504/artifact/build/artifact/wire-dev-PR2066-12504.apk)
[Download build #12509](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12509/artifact/build/artifact/wire-dev-PR2066-12509.apk)
[Download build #12511](http://192.168.120.33:8080/job/Pull%20Request%20Builder/12511/artifact/build/artifact/wire-dev-PR2066-12511.apk)

### CODE REVIEW

File: app/src/main/scala/com/waz/zclient/participants/fragments/SingleParticipantFragment.scala [hunk 1]
```diff
@@ -174,12 +178,14 @@ class SingleParticipantFragment extends FragmentHelper {
     // TODO: merge this logic with ConversationOptionsMenuController
     (for {
       createPerm <- userAccountsController.hasCreateConvPermission
-      convId     <- participantsController.conv.map(_.id)
-      remPerm    <- userAccountsController.hasRemoveConversationMemberPermission(convId)
+      conv       <- participantsController.conv
```
Comment (makingthematrix) [inline app/src/main/scala/com/waz/zclient/participants/fragments/SingleParticipantFragment.scala]: Why the change from `convId` to `conv`?

Comment (johnxnguyen) [reply app/src/main/scala/com/waz/zclient/participants/fragments/SingleParticipantFragment.scala]: Because I need `conv` to check if the other participant is a guest in the team, but maybe this is no longer needed if the spec was wrong.
